libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java

```diff
@@ -389,6 +393,8 @@ public static String getEventNameForStat(AnalyticsTracker.Stat stat) {
                 return "reader_blog_previewed";
             case READER_BLOG_UNFOLLOWED:
                 return "reader_site_unfollowed";
+            case READER_ARTICLE_VIEWED:
+                return "reader_article_viewed";
```

Comment:
I wonder can we rename this to something different, to better distinguish it from `reader_article_opened` ? Personally, I will probably have a hard time telling the difference from analytics dashboard. WDYT?

Reply:
I chose `READER_ARTICLE_VIEWED` to follow the naming convention of the other buttons (e.g. `READER_ARTICLE_LIKED`).  We could change it to `READER_ARTICLE_VIEW_TAPPED`, but then we should probably change all the buttons to `READER_ARTICLE_<BUTTON>_TAPPED` for consistency and that might make legacy analytics hard to track.  What do you think?

Reply:
I chose `READER_ARTICLE_VIEWED` to follow the naming convention of the other buttons (e.g. `READER_ARTICLE_LIKED`).  We could change it to `READER_ARTICLE_VIEW_TAPPED`, but then we should probably change all the buttons to `READER_ARTICLE_<BUTTON>_TAPPED` for consistency and that might make legacy analytics hard to track.  What do you think?

Reply:
How about READER_ARTICLE_VISITED or READER_ORIGINAL_ARTICLE_VISITED (VIEWED)? `Visited` corresponds to the actual button label (Visit), and it's easier to connect to in-app action when browsing tracks. I don't have strong opition about it, and we can allways add description to Tracks event :)